app/models/loan.rb

```diff
@@ -64,6 +70,8 @@ class Loan < ActiveRecord::Base
   attr_option_settable :status, :loan_type, :project_type, :public_level
 
 
+  attr_custom_model_linkable LOAN_CRITERIA_FIELD_SET_SYMBOL, POST_ANALYSIS_FIELD_SET_SYMBOL
```

Comment:
Fair enough.  In addition to being a convention from the java world, most of my work has been in environments where time pressures precluded the ability to maintain rigorous test cases, so I naturally look for other ways to guarantee system integrity.  I'll remove the constants.